flink-kubernetes-operator/src/main/resources/META-INF/NOTICE

```diff
@@ -70,25 +70,25 @@ This project bundles the following dependencies under the Apache Software Licens
 - org.apache.commons:commons-compress:1.21
 - org.apache.commons:commons-lang3:3.12.0
 - org.apache.commons:commons-math3:3.5
-- org.apache.flink:flink-annotations:1.14.4
-- org.apache.flink:flink-clients_2.12:1.14.4
-- org.apache.flink:flink-core:1.14.4
-- org.apache.flink:flink-file-sink-common:1.14.4
-- org.apache.flink:flink-hadoop-fs:1.14.4
-- org.apache.flink:flink-java:1.14.4
-- org.apache.flink:flink-metrics-core:1.14.4
-- org.apache.flink:flink-optimizer:1.14.4
-- org.apache.flink:flink-queryable-state-client-java:1.14.4
-- org.apache.flink:flink-rpc-akka-loader:1.14.4
-- org.apache.flink:flink-rpc-core:1.14.4
-- org.apache.flink:flink-runtime:1.14.4
-- org.apache.flink:flink-shaded-asm-7:7.1-14.0
-- org.apache.flink:flink-shaded-force-shading:14.0
-- org.apache.flink:flink-shaded-guava:30.1.1-jre-14.0
-- org.apache.flink:flink-shaded-jackson:2.12.4-14.0
-- org.apache.flink:flink-shaded-netty:4.1.65.Final-14.0
-- org.apache.flink:flink-shaded-zookeeper-3:3.4.14-14.0
-- org.apache.flink:flink-streaming-java_2.12:1.14.4
+- org.apache.flink:flink-annotations:1.15.0
```

Comment:
We might need to review the bundled transitive dependencies since we bump the Flink version.